src/app-toolkit/app-toolkit.helpers.ts

```diff
@@ -56,5 +58,7 @@ export class AppToolkitHelperRegistry {
     public readonly masterChefRewarderClaimableBalanceStrategy: MasterChefRewarderClaimableBalanceStrategy,
     @Inject(MasterChefRewarderClaimableTokenStrategy)
     public readonly masterChefRewarderClaimableTokenStrategy: MasterChefRewarderClaimableTokenStrategy,
```

Comment:
Can we rename this to `MasterChefV2ClaimableTokenStrategy` and `MasterChefV2ClaimableBalanceStrategy` for consistency? 🙏